src/org/labkey/test/tests/ExternalSchemaTest.java

```diff
@@ -270,6 +277,15 @@ public void testExternalSchemaDelete()
         String containerPath = PROJECT_NAME + "/" + FOLDER_NAME_2;
         ensureExternalSchema(containerPath);
         _schemaHelper.deleteSchema(containerPath, USER_SCHEMA_NAME);
+        assertTrue("expect audit event for schema deletion",
+                getMatchingContainerAuditEvents(containerPath, "External schema 'Test' deleted"));
+    }
+
+    boolean getMatchingContainerAuditEvents(String containerPath, String matchingComment)
+    {
+        var rows = executeSelectRowCommand("auditLog", "ContainerAuditEvent",
+                ContainerFilter.Current, containerPath, null).getRows();
+        return rows.stream().anyMatch(a-> a.get("_labkeyurl_container").toString().contains(containerPath) && a.get("comment").toString().equals(matchingComment));
     }
```

Comment:
Consider navigating to the audit page if the check fails. This will be painful to investigate if it fails.
```suggestion
    void assertContainerAuditEvents(String containerPath, String matchingComment)
    {
        var rows = executeSelectRowCommand("auditLog", "ContainerAuditEvent",
                ContainerFilter.Current, containerPath, null).getRows();
        boolean foundEvent = rows.stream().anyMatch(a-> a.get("_labkeyurl_container").toString().contains(containerPath) && a.get("comment").toString().equals(matchingComment));
        if (!foundEvent)
        {
            beginAt(WebTestHelper.buidUrl("audit", containerPath, "showAuditLog", Map.of("view", "ContainerAuditEvent")); // TODO: Add a new `ShowAuditLogPage.beginAt` method that takes the audit event type
            Assert.fail("Event not found: " + matchingComment);
        }
    }
```

Reply:
Good idea.  Thanks!
I took the liberty of wiring up that ShowAuditLogPage.beginAt method, and using it here